### DISCUSSION TITLE
Customise each row of data grid based on field value.

### DISCUSSION THREAD

**Discussion by FraserElectronics** [AvaloniaUI/Avalonia#14751] [Q&A] [answered]
I am using two data grids to show related data. I would like to highlight the rows whose values differ. Is it possible to change the colour / style of a row in the data grid based on a boolean field in the bound data?

Andy

**Comment by workgroupengineering** [comment, marked as answer]
see https://github.com/AvaloniaUI/Avalonia/discussions/13545#discussioncomment-7516089

**Reply by FraserElectronics** [reply]
Cool, so I can do this using data templates - thanks for the pointer 👍

Andy

**Reply by timunie** [reply]
DataTemplates or RowTheme / CellTheme and converter
